src/base/utils/use-knob.tsx

```diff
@@ -2,7 +2,8 @@
 import { useState } from 'react';
 import * as knobTypes from '@storybook/addon-knobs';
 
-import { manager } from '@storybook/addon-knobs/dist/registerKnobs';
+// @ts-ignore
+import { manager } from '@storybook/addon-knobs/dist/esm/registerKnobs';
```

Comment:
It would be a good idea to slowly migrate away from addon-knobs to addon-controls: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addon-knobs

Reply:
Completely agree, just a large migration at the moment.

Reply:
Yeah it's a pretty large migration especially for a project of this size.